spec/system/side_nav_bar_system_spec.rb

```diff
@@ -14,27 +14,43 @@
   it 'navigate through side nav bar' do
     login_as_user(:bob)
 
-    sidebar = page.find('#sidebar')
+    sidebar = page.find('div.side-nav-bar-teams-list')
     expect(sidebar).to be_present
 
     expect(sidebar).to have_text('team1')
     expect(sidebar).to have_text('team2')
 
+    team1_link = find('a', text: 'team1', visible: false)
+    team2_link = find('a', text: 'team2', visible: false)
+
+    team1 = teams(:team1)
+    team2 = teams(:team2)
+    folder2 = folders(:folder2)
+
+    # Click on Team and check if expands and collapses
     within(sidebar) do
-      team1_link = find('a', text: 'team1', visible: false)
-      team2_link = find('a', text: 'team2', visible: false)
+      team1_collapsed_in_sidebar?
+      folder2 false
+
+      team1_link.click
+      expect(uri).to eq "/teams/#{team1.id}"
 
-      team1 = teams(:team1)
-      team2 = teams(:team2)
-      folder2 = folders(:folder2)
+      team1_expanded_in_sidebar?
+      folder2 true
```

Comment:
you got a variable and a method that is called folder2
I think it's also a good idea to use brackets here for a method call. 
maybe also use named params here assert_folder2(shown: true)